data_steward/tools/run_basics_remediation.py

```diff
@@ -0,0 +1,255 @@
+        required=True,
+        help='Release tag for the new datasets after remediation.')
+    parser.add_argument('-l',
+                        '--console_log',
+                        dest='console_log',
+                        action='store_true',
+                        required=False,
+                        help='Log to the console as well as to a file.')
+
+    return parser.parse_args(raw_args)
+
+
+def main():
+    """
+    Read through LOGGER.info() messages for what it does.
+    """
+    args = parse_args()
+
+    tag: str = args.new_release_tag
+    project_id: str = args.project_id
+
+    dataset: str = args.source_dataset_id
+    new_dataset: str = get_new_dataset_name(dataset, tag)
+    sb_dataset: str = f"{tag}_sandbox"
+
+    pipeline_logging.configure(level=logging.INFO,
+                               add_console_handler=args.console_log)
+
+    impersonation_creds = get_impersonation_credentials(args.run_as_email,
+                                                        CDR_SCOPES)
+    client = BigQueryClient(args.project_id, credentials=impersonation_creds)
+
+    LOGGER.info(
+        f"Starting the basics remediation.\n"
+        f"This script will copy the following dataset to the new dataset with "
+        f"the new release tag {tag}, and run remediation on the new dataset. \n"
+        f"\n"
+        f"Source dataset: {dataset}. \n"
+        f"New dataset: {new_dataset}. \n"
+        f"\n"
+        f"No change will be made to the source datasets, and all the remediation "
+        f"will be sandboxed to {sb_dataset}.\n"
+        f"\n"
+        f"Steps:\n"
+        f"[1/5] Create the empty new dataset if not exist\n"
+        f"[2/5] Copy data from the source dataset to the new dataset\n"
+        f"[3/5] Create an empty sandbox dataset if not exists\n"
+        f"[4/5] Run remediation and cleaning rules on the new dataset\n"
+        f"[5/5] Run cleaning rules on the new datasets\n"
+        f"\n"
+        f"If you answer 'Y', [1/5] will start. \n"
+        f"You will need to answer 'Y' after each step completes.\n")
+
+    LOGGER.info(f"Starting [1/5] Create the empty new dataset if not exist.")
+    ask_if_continue()
+    create_dataset(client, dataset, tag)
+    LOGGER.info(f"Completed [1/5] Create the empty new dataset if not exist.\n")
+
+    LOGGER.info(f"Starting [2/5] Copy data to the new dataset.")
+    ask_if_continue()
+    copy_dataset(client, dataset, tag)
+    LOGGER.info(f"Completed [2/5] Copy data to the new dataset.\n")
+
+    LOGGER.info(
+        f"Starting [3/5] Create an empty sandbox dataset if not exists.")
+    ask_if_continue()
+    create_sandbox_dataset(client, tag)
+    LOGGER.info(
+        f"Completed [3/5] Completed an empty sandbox dataset if not exists.\n")
+
+    LOGGER.info(
+        f"Starting [4/5] Run remediation and cleaning rules on the new dataset."
+    )
+    ask_if_continue()
+    if not is_deid_dataset(new_dataset) and not is_combined_dataset(
```

Comment:
@hiro-mishima, This logic check is fine for now, but will need to be removed when we remediate the lower priority datasets.  Just putting the comment here as a placeholder.

Reply:
Thanks! I removed this check at the PR for DC-3079: https://github.com/all-of-us/curation/pull/1489/commits/4b12430e03556fc106b1435f871ffd2f94aaa35d#diff-86e18cafe179274064a9a88590e2ce09cf6204860c5ad9d0b4db4ff2135ad5c0L223